shivyc/ctypes.py

```diff
@@ -266,15 +274,15 @@ def is_function(self):
         return True
 
 
-class StructCType(CType):
-    """Represents a struct ctype.
+class _UnionStructCType(CType):
+    """Base class for struct and union C types.
 
-    tag - Name of the struct as a string, or None if it's anonymous
+    tag - Name of the variable as a string, or None if it's anonymous
```

Comment:
Nitpicky: the tag isn't the name of the variable per se, it's the name of the type. E.g. in the following:
```
struct S {
  int a;
} var;
```
the tag would be `S` although the variable name is `var`.